R/content.R

```diff
@@ -978,16 +979,20 @@ get_content_permissions <- function(content, add_owner = TRUE) {
 #' }
 #' 
 #' @export
-content_render <- function(content) {
+content_render <- function(content, variant_key = NULL) {
```

Comment:
Yeah, I realize, but I included the `if` statement because I would rather have `NULL` represent the "no variant key provided" case than `"default"` — that feels more… idiomatically expected to me.